Source/Hypermedia.Client/Resolver/NoLinkCache.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace Bluehands.Hypermedia.Client.Resolver
+{
+    public class NoLinkCache<T> : ILinkHcoCache<T>
+    {
+        private NoLinkCache()
+        {
+
+        }
+
+        public static NoLinkCache<T> Instance { get; } = new NoLinkCache<T>();
+
+        public bool TryGetValue(Uri uri, out CacheEntry<T> entry)
+        {
+            entry = null;
```

Comment:
```suggestion
            entry = new CacheEntry<T>();
```
For changing CacheEntry from class to struct